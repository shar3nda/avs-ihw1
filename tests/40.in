10
6 3 -1 6 9 5 8 5 2 5