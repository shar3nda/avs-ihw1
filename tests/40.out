-1 10 1 -2 10 -1 10 9 0